src/CCInput.js

```diff
@@ -34,6 +34,7 @@ export default class CCInput extends Component {
     onChange: PropTypes.func,
     onBecomeEmpty: PropTypes.func,
     onBecomeValid: PropTypes.func,
+    additionalInputProps: TextInput.propTypes,
```

Comment:
This'll need to be `PropTypes.shape(TextInput.propTypes)` or React will complain that it's not a valid propTypes